crates/iota/src/iota_commands.rs

```diff
@@ -363,6 +363,8 @@ pub enum IotaCommand {
     /// Invoke Iota's move-analyzer via CLI
     #[clap(name = "analyzer", hide = true)]
     Analyzer,
+    #[cfg(feature = "gen-completions")]
```

Comment:
Please add a doc comment to it appears in the CLI usage/help

Reply:
Oh good call, thanks

Reply:
Done, and I also removed the list of shells as clap already prints like this:

```
  -s, --shell <SHELL>      The shell for which completions will be generated. By default all will be generated [possible values: bash, elvish, fish,
                           nushell, powershell, zsh]
```